pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -569,6 +569,66 @@ func (c *Cluster) handleRemovedNodes(config *provisionConfig) {
 	logger.Infof("done processing removed nodes")
 }
 
+func (c *Cluster) cleanupTimeoutNode(selector string, config *provisionConfig, nodeName, crushName string) {
```

Comment:
Is the purpose of this method to cleanup the status info so the orchestration can continue instead of being stuck on the nodes that aren't responding? In this case, it may be sufficient to just delete the status configmap.

Reply:
Yes. OK, I will change the method to just delete status configmap